### ISSUE TITLE
Extra line breaks added in empty script tags

### ISSUE THREAD

**Issue by brennaveen** [mblode/vscode-twig-language-2#33]
I know you said in #29 that you were ignoring script tags, but when I have an empty script tag the formatter is taking it from:

```
    <script src="https://cdn.datatables.net/v/bs4/jszip-2.5.0/dt-1.10.18/b-1.5.2/b-html5-1.5.2/b-print-1.5.2/datatables.min.js"></script>
```

to

```
    <script src="https://cdn.datatables.net/v/bs4/jszip-2.5.0/dt-1.10.18/b-1.5.2/b-html5-1.5.2/b-print-1.5.2/datatables.min.js">



</script>
```

Obviously, this doesn't break anything, but it does make the code look odd.

**Comment by mblode** [comment]
This has now been fixed in the latest update 😄 But please let me know if you have any other issues.

**Comment by mblode** [comment]
Fixed empty script block formatting in 0.12.0 and added a regression test.
